src/dealDashboard/dealDiscussion/discussionThread/discussionThread.ts

```diff
@@ -352,6 +338,25 @@ export class DiscussionThread {
     }
   }
 
+  private addAuthorToThreadProfiles(comment: IComment): void {
+    if (!this.threadProfiles[comment.author]) {
+      this.isLoading[comment.author] = true;
+      if (comment.authorENS /* author has ENS name */) {
+        this.threadProfiles[comment.author] = {
+          name: comment.authorENS,
+          address: comment.author,
+          image: "",
+        };
+      } else {
+        /* required for replies */
+        this.discussionsService.loadProfile(comment.author).then(profile => {
+          this.threadProfiles[comment.author] = profile;
+          this.isLoading[comment.author] = false;
```

Comment:
Wondering why it's enough to have this change to `= false` in the `else` statement only?

loading `= true` is set before the `if` in line 343.
So I expect `= false` at the end of the `if`, ie. after line 356

@giladt

Reply:
We want to set the loading to false only after the `loadProfile` promise is fulfilled. Otherwise, it will be set to `false` immediately.

But, you are right- 
- Setting `isLoading[comment.author] = true` can be moved into the `else` statement, since it's the only async operation, and 
- the setting to `false` should probably be better inside a `.finally()` statement... 
- (Applies also for line 240...)
@hiaux0

Reply:
Looks like a minor improvement then.
Will skip